Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/theme.css

```diff
@@ -0,0 +1,16 @@
+/*--------------------------------------------------------------------------------
+These default CSS variables can be overridden and customized by creating a file, 
+containing a copy of the below, called "PersonaBarTheme.css" and placing it in 
+the "Portals/_default" directory. If the custom CSS file exists, it will be loaded 
+instead of this one.
+---------------------------------------------------------------------------------*/
+:root{
+    --dnn-color-pb-menu-background: #0e2936;
+    --dnn-color-pb-menu-background-hover: #0b1c24;
+    --dnn-color-pb-menu-icon: #3c7a9a;
+    --dnn-color-pb-menu-icon-background: #0b1c24;
+    --dnn-color-pb-menu-divider: #1e485e;
+    --dnn-color-pb-menu-text-highlight: #3c7a9a;

```

Comment:
Same question here for colors etc. being able to change background without foreground colors is limiting.

Reply:
This achieves all the styling changes as it relates to the initial goals for this solution.  This also handles "foreground colors".  They just aren't called "foreground".

Reply:
IMO, this is just for the actual menu icons and background, if we get into like text foreground colors, then we open a big can of worms into the actual module panels which can have also any number of 3rd party modules. Not saying it's not a good idea, but it's a much much bigger ticket to do right. I would love to do that but in a separate way for the panels (different customization file) later on.